mlir/test/fusion/e2e/mixr-gemm/mixr-gemm-add.e2e.mlir

```diff
@@ -1,11 +1,20 @@
 // RUN: rocmlir-opt -migraphx-to-tosa %s | rocmlir-driver -host-pipeline highlevel | rocmlir-gen -ph -print-results -rand none - | rocmlir-driver -c  | mlir-rocm-runner --shared-libs=%linalg_test_lib_dir/libmlir_rocm_runtime%shlibext,%conv_validation_wrapper_library_dir/libconv-validation-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext --entry-point-result=void | FileCheck %s
-
+// RUN: rocmlir-opt -migraphx-to-tosa %s | rocmlir-driver -host-pipeline partition,highlevel -targets gfx908,gfx90a | rocmlir-gen -ph -print-results -rand none -fut dot_add --verifier clone - | rocmlir-driver -host-pipeline xmodel -kernel-pipeline full -triple amdgcn-amd-amdhsa -targets gfx908,gfx90a | xmir-runner --shared-libs=%linalg_test_lib_dir/libmlir_rocm_runtime%shlibext,%conv_validation_wrapper_library_dir/libconv-validation-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext --entry-point-result=void | FileCheck %s --check-prefix=CLONE
 module {
   // CHECK:  [5,     5,     5],
   // CHECK-NEXT: [5,     5,     5],
   // CHECK-NEXT: [5,     5,     5],
   // CHECK-NEXT: [5,     5,     5],
   // CHECK-NEXT: [5,     5,     5]
+
+  // CLONE: [1 1 1]
+  // CLONE-NEXT: Unranked Memref base
```

Comment:
Let's use '-rand 1 -rand_type float' for all these so we are sure it is working. And you can just check for '[1 1 1]' then.

Reply:
Done, for all but tosa-to-rock-rsqrt.e2e.mlir.  For that one, the new random options caused all the result values to be NaN, I think because intermediate values were negative.  Left it with "-rand none".